debian/copyright

```diff
@@ -1,7 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: nymea-plugins
+Upstream-Name: nymea-plugins-zigbee
 Upstream-Contact: nymea GmbH <contact@nymea.io>
-Copyright: 2014-2017, guh GmbH
+Copyright: 2014-2022, guh GmbH
```

Comment:
nymea GmbH

Reply:
This was actually an issue in the base branch. Fixed it there and rebased this properly